catroid/src/main/java/org/catrobat/catroid/ProjectManager.java

```diff
@@ -160,6 +160,9 @@ public void loadProject(String projectName, Context context) throws LoadingProje
 		if (project.getCatrobatLanguageVersion() == 0.998f) {
 			project.setCatrobatLanguageVersion(0.999f);
 		}
+		if (project.getCatrobatLanguageVersion() == 0.998f) {
+			project.setCatrobatLanguageVersion(0.999f);
+		}
```

Comment:
This `if` is identical to the above one and will always evaluate to false :smile_cat:

Reply:
whoopsie.. ill fix that right away